frontend/src/components/DiscussionList.vue

```diff
@@ -68,14 +68,12 @@
                   <span :class="filters ? '' : 'hidden sm:inline'">
                     {{ user.full_name }}
                   </span>
-                  <template v-if="!filters || !filters.project">
```

Comment:
This was added so that project name is not visible if discussion list is being viewed inside project.
For e.g., `/g/all-hands/projects/12/discussions/`

Reply:
We are passing list of active projects to exclude discussions from archived projects on the discussions page. This condition is preventing display of the respective project names. Can it be passed as a prop instead, like displayProject?